src/NzbDrone.Core/HealthCheck/Checks/ProxyCheck.cs

```diff
@@ -33,7 +33,6 @@ public override HealthCheck Check()
                 }
 
                 var request = new HttpRequestBuilder("https://sonarr.tv").Build("/");
```

Comment:
@markus101  maybe idea to replace this with `http(s)://services.sonarr.tv/ping`?